File: src/com/cattailsw/nanidroid/KeroView.java
package com.cattailsw.nanidroid;

import android.content.Context;
import android.util.AttributeSet;

public class KeroView extends SakuraView {
    public KeroView(Context ctx){
	super(ctx);
    }
    public KeroView(Context ctx, AttributeSet attr){
	super(ctx, attr);
    }

    public KeroView(Context ctx, AttributeSet attr, int defStyle){
	super(ctx, attr, defStyle);
    }

    public void changeSurface(String surfaceid){
	currentSurface = mgr.getKeroSurface(surfaceid);
    }


}

File: src/com/cattailsw/nanidroid/KeroView.kt
package com.cattailsw.nanidroid

import android.content.Context
import android.util.AttributeSet

/** Kero-specific surface resolver for the shared retained surface view. */
open class KeroView @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null,
    defStyle: Int = 0,
) : SakuraView(context, attrs, defStyle) {
    protected override fun loadSurface(surfaceId: String) {
        currentSurface = mgr!!.getKeroSurface(surfaceId)
    }
}
